root/init-core

```diff
@@ -54,4 +54,8 @@ umask 0002
 
 service php7.3-fpm start
 service nginx start
+## start cron
```

Comment:
This will probably be needed in root/init-admin also.

Reply:
Thx for review )